### ISSUE TITLE
Meteor 1.5-rc.7 to rc-10 error dynamic-import

### ISSUE THREAD

**Issue by ixdi** [meteor/meteor#8726]
When  we upgrade our software from 1.5-rc.6 to rc.7 an error appears. The version rc.6 is working fine, we are testing it since since it came out. The rc.10 is also failing. It is happening with the rest of rc from 7 to 10.

```js
dynamic-import.js?hash=0fc75d9…:183 Uncaught (in promise) TypeError: Object.defineProperty called on non-object
    at Function.defineProperty (<anonymous>)
    at getNamespace (dynamic-import.js?hash=0fc75d9…:183)
    at dynamic-import.js?hash=0fc75d9…:42
    at meteor.js?hash=27829e9…:1105
    at <anonymous>
```
Thanks

**Comment by benjamn** [comment]
I believe this will be fixed once `1.5-rc.11` is published, thanks to [this commit](https://github.com/meteor/meteor/pull/8327/commits/cd4fc028f8b4bf415e6520b764b137009876ab73).

**Comment by ixdi** [comment]
Yes, works well with rc-12, thank you!
